### PR TITLE
README: Improve wording, formatting, links, headers

### PR DESCRIPTION
**Pull request by jzaefferer**
At this point dropping "an attempt" in the opening paragraph seems well justified. The rest of the wording there is still reasonably cautious.

The rest should be even less problematic. Unless there's a reason to keep links in plaintext.

Closes #93

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
-If you maintain a JavaScript library that has UI components, we'd love for you to join our efforts. We'd be happy to hear your feedback and work through updates of your project(s) to support CSS Framework. If you have the time and interest to help define the standards and improve on our implementation, that's even better! Check out the [contribution guidelines](https://github.com/jquery/css-chassis/blob/master/CONTRIBUTING.md) for more information.
+If you maintain a JavaScript library that has UI components, we'd love for you to join our efforts. We'd be happy to hear your feedback and work through updates of your project(s) to support Chassis. If you have the time and interest to help define the standards and improve on our implementation, that's even better! Check out the [contribution guidelines](https://github.com/jquery/css-chassis/blob/master/CONTRIBUTING.md) for more information.
 
 
 ## Getting started
 
 ### Prerequisites
 
-Node.js and npm should be installed. [[installer]](http://nodejs.org/download/)
+Node.js and npm [should be installed](http://nodejs.org/download/).
```
Comment (markelog) [inline README.md]: It seems it would be nice to add `grunt` package as global dependency in here

File: README.md [hunk 2]
```diff
@@ -1,28 +1,27 @@
-Chassis is an attempt at creating open standards designed for CSS libraries, JavaScript UI libraries, and web developers in general. This project will define standards for markup and class names for common widgets and patterns, as well as an implementation of those standards. Just like DOM libraries can build upon [Sizzle](https://github.com/jquery/sizzle), we hope UI libraries will build upon this.
+Chassis is creating open standards designed for CSS libraries, JavaScript UI libraries, and web developers in general. This project will define standards for markup and class names for common widgets and patterns, as well as an implementation of those standards. Similar to how DOM libraries can build upon [Sizzle](https://github.com/jquery/sizzle), we hope UI libraries will build upon this project.
```
Comment (sfrisk) [inline README.md]: I agree, taking "an attempt at" out of this paragraph makes sense.

File: README.md (hunk 1 above)
Comment (sfrisk) [inline README.md]: :+1:

**Comment by sfrisk** [comment]
Overall looks good to me.

File: README.md (hunk 1 above)
Comment (jzaefferer) [inline README.md]: I can't update this until next week, feel free to land this and add any remaining enhancements as needed.

Comment (jzaefferer) [inline README.md]: Updated, now mentions `grunt-cli` as well.

**Comment by jzaefferer** [comment]
I think this is good to land, isn't it? The only non-outdated review comment was basically a :+1:

**Comment by sfrisk** [comment]
Yeah, landing this tonight, was just giving everyone one last chance to say
yay/nay.

On Tue, Sep 15, 2015 at 3:57 PM, Jörn Zaefferer <notifications@github.com>
wrote:

> I think this is good to land, isn't it? The only non-outdated review
> comment was basically a [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jquery/css-chassis/pull/115#issuecomment-140519060>.
>



-- 
Sarah Frisk
sarahfrisk.com

Twitter: twitter.com/SarahFrisk
LinkedIn: linkedin.com/in/sarahfrisk
Github: github.com/sfrisk

**Pull request closed without merging**